test/unit/custom_spec.js

```diff
@@ -107,3 +107,129 @@ describe("custom canvas rendering", function () {
       .catch(done.fail);
   });
 });
+
+describe("alternate document context", function () {
+  const global = typeof globalThis !== "undefined" ? globalThis : window;
```

Comment:
Sorry, but this isn't correct. The whole point of `globalThis` is that it's always defined, and in a "normal" browser context `globalThis === window`.

Hence please remove this line, and change `global` to `globalThis` below.

Reply:
This was my first inclination. I didn't know how far back you wanted to support. 

Sorry, for the irritation￼.